activerecord/lib/active_record/inheritance.rb

```diff
@@ -105,21 +105,45 @@ def base_class
         end
       end
 
-      # Set this to true if this is an abstract class (see <tt>abstract_class?</tt>).
-      # If you are using inheritance with ActiveRecord and don't want child classes
-      # to utilize the implied STI table name of the parent class, this will need to be true.
-      # For example, given the following:
+      # Set this to +true+ if this is an abstract class (see
+      # <tt>abstract_class?</tt>).
+      # If you are using inheritance with Active Record and don't want a class
+      # considered part of an STI hierarchy, this must be true.
```

Comment:
I feel like this sentence is grammatically incorrect somewhat. What do you think about:

>If you are using inheritance with Active Record and don't want a class **to be** considered **as** part of **the** STI hierarchy, **you should set this to +true**.